services/federation-upload/src/test/resources/application.yaml

```diff
@@ -0,0 +1,27 @@
+---
+logging:
+  level:
+    org:
+      springframework: off
+    root: off
+
+services:
+  upload:
+    privatekey: ${VAULT_FILESIGNING_SECRET:file:/Users/i850193/workspaces/cwa/cwa-server/docker-compose-test-secrets/private.pem}
```

Comment:
the default of the property can be removed or pointed to a classpath pem fie src/test/resources

Reply:
This seems to be the old file its path is under `federation-upload`. I must have accidentally pushed it. The proper one `upload/src/test/resources/application.yaml` points to a classpath file on the resources folder.